Source/Core/Logging/FramesLogEvent.swift

```diff
@@ -63,6 +68,9 @@ enum FramesLogEvent: Equatable {
         case .paymentFormPresented,
              .billingFormPresented:
             return [:]
+        case let .checkoutAPIServiceInitialised(environment):
+            let environmentString = environment.rawValue == "live" ? "production" : environment.rawValue
```

Comment:
Do we know why we have both live and production ? and can we sync this as just being production in future.

Reply:
There is currently only live in the SDK. This is specifically setting it to production for this log. When we move to 4.0 we should replace all instances of live with production